src/scss/components/password-input/_index.scss

```diff
@@ -0,0 +1,104 @@
+@import "components/form/config";
+@import "mixins";
+
+hx-password-control {
+  $ctrl-columns: auto 1fr auto;
+  $ctrl-rows: auto auto;
+  $ctrl-row-align: center;
+
+  -ms-grid-columns: $ctrl-columns;
+  -ms-grid-rows: $ctrl-rows;
+  align-items: $ctrl-row-align;
+  display: -ms-grid;
+  display: grid;
+  grid-template-areas:
+    "label  label   label"
+    "prefix control suffix";
+  grid-template-columns: $ctrl-columns;
+  grid-template-rows: $ctrl-rows;
+
+  // blacklist
+  > * {
+    display: none;
+  }
+
+  // whitelist
+  > label,
+  > input[type="password"],
+  > .hxPrefix,
+  > .hxSuffix {
+    display: block;
+  }
+
+  > label {
+    $justify: start;
+
+    -ms-grid-column-align: $justify;
+    -ms-grid-column-span: 3;
+    -ms-grid-column: 1;
+    -ms-grid-row: 1;
+    grid-area: label;
+    justify-self: $justify;
+  }
+
+  > input[type="password"] {
+    -ms-grid-column: 2;
+    -ms-grid-row: 2;
+    grid-area: control;
+    height: 2rem;
+    min-width: 8rem; // ~16 chars
+    padding: 0 0.75rem;
+  }
+}
+
+// ----- PRISTINE ----------
+hx-password-control {
+  > label {
+    font-size: 14px;
+    line-height: 1.5;
+    color: $gray-800;
+  }
+
+  > p {
+    @include hxHelpText();
+  }
```

Comment:
I just realized we have default styles for `Help Text` 🤦‍♂ 

Please update block:

```scss
> p {
  @include hxHelpText();

  .hxHelpText { . // TODO: Verify this class is needed.
    @include hxHelpText();

  .hxErrorText {
    color: $red-900;
    font-size: 0.75rem;
  }
```

Reply:
Changes have been done, please review.